src/content/app/new-species-selector/types/speciesSearchMatch.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type SpeciesSearchMatch = {
+  genome_id: string;
+  common_name: string | null;
+  scientific_name: string;
+  type: {
+    kind: string; // e.g. "population"
+    value: string; // e.g. "European"
```

Comment:
Do we have predefined values for these two properties rather than random ones?

Reply:
The list of "kinds" must be small (but I do not have it); the list of values, large and open. I think, `string` is an appropriate representation for both.

Reply:
Okay so let's change the type of `kind`, when we know what the values for it, later and ignore `value`.